src/Option.php

```diff
@@ -62,26 +62,18 @@ public function set($key, $value = null)
         $keys = is_array($key) ? $key : [$key => $value];
 
         foreach ($keys as $key => $value) {
-            $option = self::firstOrNew(['key' => $key]);
-
-            $option->value = $value;
-
-            $option->save();
+            self::firstOrCreate(['key' => $key], ['value' => $value]);
```

Comment:
This will not work when updating an existing option

Reply:
You are correct, I used the wrong function.
I have created an edit where I replaced the function with `updateOrCreate`.

Reply:
Awesome!